docs/aws/Authentication.rst

```diff
@@ -38,5 +38,5 @@ IoT policy assigned to the Cognito Identity:
 .. literalinclude:: ./iot-policy.json
   :language: JSON
 
-The two specified policies will be ANDed and only the least privilege of the combined privilege is granted.
+The two specified policies are combined and only the least privilege of the combined privilege is granted.
```

Comment:
This changes the meaning of the sentence, because it removes the information that logical AND is used to combine them.

```suggestion
The two specified policies are combined using logical AND and only the least privilege of the combined privilege is granted.
```